### PR TITLE
[1.10] Skip restablish vtep

### PR DESCRIPTION
**Pull request by mainred**
Reconfiging mac address of an interface will reset its static ip neighbours.
Thus, static ip neighbours of vtep interface will be removed,
if dcos_overlay is restarted, which results in traffic break.

Fix this by skipping configuring an existing vtep interface.
Also we delete and create the vtep interface if its configuration is not up to date.

Close-Bug: [DCOS_OSS-3855](https://jira.mesosphere.com/browse/DCOS_OSS-3855)

### CODE REVIEW

**Comment by mainred** [comment]
@urbanserj @GoelDeepak  PTAL

**Review by GoelDeepak — APPROVED** [review]

File: apps/dcos_overlay/testdata/vtep_link.data [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[
```
Comment (urbanserj) [inline apps/dcos_overlay/testdata/vtep_link.data]: Please remove this file.

**Review by urbanserj — CHANGES_REQUESTED** [review]

File: apps/dcos_overlay/testdata/vtep_link.data (hunk 1 above)
Comment (mainred) [reply apps/dcos_overlay/testdata/vtep_link.data]: done

**Review by urbanserj — APPROVED** [review]

**Comment by urbanserj** [comment]
@mainred it seems like your PR broke master branch in this repo. could you please take a look?

https://circleci.com/gh/dcos/navstar/510
https://circleci.com/gh/dcos/navstar/511
https://circleci.com/gh/dcos/navstar/512

**Comment by mainred** [comment]
I am on it

**Pull request merged**